release/mac/dataloader.command

```diff
@@ -1,14 +1,18 @@
 #!/bin/bash
+DATALOADER_VERSION="@@FULL_VERSION@@"
+DATALOADER_SHORT_VERSION=$(echo ${DATALOADER_VERSION} | cut -d'.' -f 1)
+DATALOADER_UBER_JAR_NAME="dataloader-${DATALOADER_VERSION}-uber.jar"
+MIN_JAVA_VERSION=@@MIN_JAVA_VERSION@@
 
 echo ""
 echo "*************************************************************************"
 echo "**            ___  ____ ___ ____   _    ____ ____ ___  ____ ____       **"
```

Comment:
Minor: we may want to use multiple tabs instead of spaces that way whenever lines are changed it renders neatly
ie) \t

Reply:
Yes, I need to figure out a way to address it in IDE or a do a workaround by running a script that replaces tabs with spaces before checking in.